tests/edit_inputs.sh

```diff
@@ -10,9 +10,14 @@ function edit_ice_in {
   jday=$(date -d "${SYEAR}-${SMONTH}-${SDAY} ${SHOUR}:00:00" +%j)
   istep0=$(( ((10#$jday-1)*86400 + 10#$SHOUR*3600) / DT_CICE ))
 
+  CICEGRID="grid_cice_NEMS_mx"$OCNRES".nc"
```

Comment:
Why are CICEGRID and CICEMASK defined here? I see they are defined in default_vars.sh.  Should any overriding be done in individual tests/tests/<test-name> files?

Reply:
Yes, I think you are right. 

The other line I was unsure about (even though it works) is setting ICERES in the fv3_conf/*.IN files. Unfortunately we have files with both "025" and "0.25" as resolution identifiers. Is there a better way (or place) to set that variable?

Reply:
Here is what I think -- this will have to be tested. I leave it up to you to decide:
- Delete CICEGRID and CICEMASK in default_vars
- Move CICEGRID and CICEMASK defined in edit_inputs to `fv3_conf/*.IN` (`grid_cice_NEMS_mx@[OCNRES].nc`)
- If ICRES always depends on OCNRES, maybe replace ICERES in default_vars with ICERES in `fv3_conf/*.IN`, and remove it from `fv3_conf/*.IN`

Reply:
If you want to test these out, let me know. I can clone your branch and work on them.

Reply:
That would be a huge help Minsuk, thanks. 

ICERES and OCNRES will always represent the same resolution since they're always on the same grid. I can envision: mx025, 0.25
mx050, 0.50
mx100, 1.00